koku/api/report/test/ocp/tests_views.py

```diff
@@ -794,6 +794,61 @@ def test_execute_query_group_by_project(self):
         for entry in data.get('data', []):
             for project in entry.get('projects', []):
                 self.assertEqual(project.get('project'), project_of_interest)
+                values = project.get('values', [])
+                for value in values:
+                    self.assertIn('cluster_id', value)
+                    self.assertIn('cluster_alias', value)
+                print(project)
+
+    def test_execute_query_group_by_project_duplicate_projects(self):
+        """Test that same-named projects across clusters are accounted for."""
+        data_config = {'namespaces': ['project_one', 'project_two']}
+        project_of_interest = data_config['namespaces'][0]
+        data_generator = OCPReportDataGenerator(self.tenant)
+        data_generator.add_data_to_tenant(**data_config)
+        data_generator.add_data_to_tenant(**data_config)
+
+        url = reverse('reports-openshift-cpu')
+        client = APIClient()
+        params = {'group_by[project]': project_of_interest}
+
+        url = url + '?' + urlencode(params, quote_via=quote_plus)
+        response = client.get(url, **self.headers)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Please use `rest_framework.status` to specify status codes. It helps save time so I don't have to go look up the distinction between a 400, 401, and 403. :slightly_smiling_face:

Reply:
2️⃣  0️⃣ 0️⃣ 🆗